src/components/Footer/Footer.jsx

```diff
@@ -1,8 +1,144 @@
 
 function Footer() {
-  return <h1>footer</h1>;
+  return (
+    <div className="footerWrapper">
+      <div className="footerTop">
+        <div>
+          <Link to="/item-list" className="footerTextWhite">
+            매장안내
+          </Link>
+          <Link to="/item-list" className="footerTextWhite">
+            나이키 저널
+          </Link>
+          <Link to="/login" className="footerTextWhite">
+            로그인
+          </Link>
+          <Link to="/sign-in" className="footerTextWhite">
+            멤버 가입
+          </Link>
+        </div>
+        <div>
+          <Link to="/item-list" className="footerTextWhite">
+            고객센터
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            080-022-0182
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            주문/결제
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            배송
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            주문배송조회
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            멤버쉽 혜택/서비스
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            공지사항
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            1:1 채팅 문의
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            이용약관
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            개인정보처리방침
+          </Link>
+        </div>
+        <div>
+          <Link to="/item-list" className="footerTextWhite">
+            ABOUT NIKE
+          </Link>
+          <Link to="/item-list" className="footerTextGrey">
+            나이키에 대하여
+          </Link>
+        </div>
+        <div>
+          <Link to="/item-list" className="footerTextWhite">
+            SOCIAL
+          </Link>
+          <div className="footerTopIconBox">
+            <a href="https://twitter.com/nike">
+              <i className="fa-brands fa-twitter" />
+            </a>
+            <a href="https://ko-kr.facebook.com/nike/">
+              <i className="fa-brands fa-square-facebook" />
+            </a>
+            <a href="https://www.youtube.com/user/koreanike/videos?app=desktop">
+              <i className="fa-brands fa-youtube" />
+            </a>
+            <a href="https://www.instagram.com/nike/">
+              <i className="fa-brands fa-square-instagram" />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div className="footerCenter">
+        <div className="footerCenterBox">
+          <Link to="item-list" className="footerTextWhite">
+            대한민국
+          </Link>
+          <p className="footerTextGrey">
+            ⓒ 2022 Nike, Inc. All Rights Reserved
+          </p>
+        </div>
+        <div className="footerCenterBox">
+          <p className="footerTextGrey">이용약관</p>
```

Comment:
```suggestion
          <p className="defaultText">이용약관</p>
```
전반적으로 footer라는 이름을 사용해 특정해 주신거는 좋지만
이렇게 클래스네임을 관리하다보면 차후에는 점점 길어질 수 밖에 없습니다.
그래서 맨 처음 부모 컴포넌트에 footer라는 이름을 지정 했다면
그 아래 태그들에는 footer를 제거해도 충분히 관리할 수 있을 것입니다.

그리고 해당 요소가 어떤 역할을 하는지 클래스네임으로 구현하기 때문에
textGrey보다 더 직관적인 defaultText가 직관적인 이름이라고 생각이 됩니다.

만약 text는 그대로인데 색만 변경하고 싶다면
grey라는 색에 해당하는 스타일을 만들어 두고 동적으로 적용하는 방법을 사용하면 되겠습니다!

Reply:
수정했습니다!